run-test.py

```diff
@@ -751,9 +755,41 @@ def test_stop_start_heavy(heavy_pod, pod_ips, restore_from_backup = False):
     ok = run_benchmark(pod_ips, extra_args='-m --members-file=' + MEMBERS_FILE)
     check_benchmark(ok)
 
-    info("==== start/stop heavy at pod #"+str(heavy_pod)+" passed! ====")
+    info("==== start/stop heavy at pod #"+str(heavy_pod)+(" with restore from backup" if restore_from_backup else "")+" passed! ====")
     stop_test()
 
+def test_kill_backupmanager(heavy_pod, pod_ips, restore_from_backup = False):
+    start_test("test_kill_backupmanager" + ("_restore_from_backup" if restore_from_backup else ""))
+    info("==== kill backupmanager "+("with restore from backup " if restore_from_backup else "")+"test started ====")
+    alive = wait_until_insolar_is_alive(pod_ips, NODES, step="before-killing-backupmanager")
+    check_alive(alive)
+
+    info("Running benchmark and trying to kill backupmanager on pod #"+str(heavy_pod))
+    ssh(heavy_pod, "tmux new-session -d -s backupmanager-killer " +
+        """\\"while true; do killall -9 -r backupmanager; sleep 0.1; done""" +
+        """; bash\\" """)
+    ok = run_benchmark(pod_ips, extra_args='-s --members-file=' + MEMBERS_FILE, r=100, timeout=100)
```

Comment:
Why save members to file, if you don't use it after?

Reply:
OK, I'll fix this.